src/native/libs/build-native.sh

```diff
@@ -70,14 +70,7 @@ else
     fi
 fi
 
-if [[ "$__TargetOS" == OSX ]]; then
-    # set default OSX deployment target
-    if [[ "$__BuildArch" == x64 ]]; then
-        __CMakeArgs="-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 $__CMakeArgs"
```

Comment:
Can we delete all occurrences of `CMAKE_OSX_DEPLOYMENT_TARGET` from this file?
```sh
$ git grep -n CMAKE_OSX_DEPLOYMENT_TARGET :/src/native
src/native/libs/build-native.sh:76:        __CMakeArgs="-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 $__CMakeArgs"
src/native/libs/build-native.sh:78:        __CMakeArgs="-DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 $__CMakeArgs"
src/native/libs/build-native.sh:86:    __CMakeArgs="-DCMAKE_SYSTEM_NAME=iOS -DCMAKE_OSX_SYSROOT=iphonesimulator -DCMAKE_OSX_DEPLOYMENT_TARGET=10.0 $__CMakeArgs"
src/native/libs/build-native.sh:100:    __CMakeArgs="-DCMAKE_SYSTEM_NAME=iOS -DCMAKE_OSX_SYSROOT=iphoneos -DCMAKE_OSX_DEPLOYMENT_TARGET=10.0 $__CMakeArgs"
src/native/libs/build-native.sh:112:    __CMakeArgs="-DCMAKE_SYSTEM_NAME=tvOS -DCMAKE_OSX_SYSROOT=appletvsimulator -DCMAKE_OSX_DEPLOYMENT_TARGET=10.0 $__CMakeArgs"
src/native/libs/build-native.sh:124:    __CMakeArgs="-DCMAKE_SYSTEM_NAME=tvOS -DCMAKE_OSX_SYSROOT=appletvos -DCMAKE_OSX_DEPLOYMENT_TARGET=10.0 $__CMakeArgs"
```

Reply:
no, the ones from L86-124 are for iOS/tvOS and are still needed. we could move them to configurecompiler.cmake though.